gradoop-flink/src/test/java/org/gradoop/flink/model/impl/operators/equality/EqualityTest.java

```diff
@@ -227,10 +236,11 @@ public void testGraphEqualityByElementIds() throws Exception {
   public void testGraphEqualityByElementData() throws Exception {
     FlinkAsciiGraphLoader loader = getTestGraphLoader();
 
-    GraphEquality equality = new GraphEquality(
-      new GraphHeadToEmptyString(),
-      new VertexToDataString(),
-      new EdgeToDataString(),
+    GraphEquality<EPGMGraphHead, EPGMVertex, EPGMEdge, LogicalGraph, GraphCollection> equality =
+      new GraphEquality<>(
```

Comment:
Indentation: arguments to GraphEquality constructor should have more indentation.